### PR TITLE
PR Limit the length of the ANSI codes inputs

### PR DESCRIPTION
**Pull request by FrogletApps**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -49,7 +49,7 @@
       return tag
     }
 
-    return str.replace(/\[(\d+;)?(\d+)*m/g, function(match, b1, b2) {
+    return str.replace(/\[(\d{1,3};)?(\d{1,3})?m/g, function(match, b1, b2) {
```
Comment (toufik-airane) [inline lib/index.js:52]: How did you come with a pattern of 1 to 3 length restriction?
Are you able to provide user acceptance testing to validate that there will be no breaking change?

**Review by toufik-airane — COMMENTED** [review]
Dear @FrogletApps,

Thank you for your contribution but I've further questions regarding the content of the material.

1. Would you mind describing what did you do?
2. How did you come up with the solution?
3. Are you able to prove the successful mitigation?

Cheers.

**Review by JamieSlome — APPROVED** [review]
LGTM 👍

**Comment by huntr-helper** [comment]
Congratulations @FrogletApps - your fix has been selected! :tada:<br><br>Thanks for being part of the community & helping secure the world\'s open source code.<br>If you have any questions, please respond in the comments section. Your bounty is on its way - keep hunting!

**Pull request merged**